subprojects/core/src/main/java/org/gradle/api/internal/tasks/userinput/BuildScanUserInputHandler.java

```diff
@@ -18,8 +18,10 @@
 
 import org.gradle.internal.scan.UsedByScanPlugin;
 
+import javax.annotation.Nullable;
+
 @UsedByScanPlugin
 public interface BuildScanUserInputHandler {
-
+    @Nullable
```

Comment:
❓ Why does the implementation `getOrNull`?  Why not `getOrElse(false)` and remove the need for nullability?

Reply:
I don't know. I just added `@Nullable` to reflect the API's behavior. I'm guessing it's because the scans plugin needs to know the difference between "the user explicitly answered 'no'" and "did not get an answer from the user".